scripts/api-ping.ts

```
@@ -0,0 +1,82 @@
+		);
+		saveDb();
+		console.log(
+			`[${timestamp}] Error: ${errorMessage}, Duration: ${duration.toFixed(2)}ms`
+		);
+	}
+}
+
+async function main(): Promise<void> {
+	await initDb();
+
+	console.log(`Starting API ping to ${API_URL} every ${INTERVAL_MS}ms`);
```

ESLint

[error] Unexpected console statement.

(no-console)